droopler.install

```diff
@@ -527,18 +530,18 @@ function droopler_update_8103() {
     $configs = [
       'd_product' => [
         'field.field.node.d_product.field_page_section' => '',
-        'core.entity_view_display.node.d_product.d_small_box' => '', // @todo: Check if this should be here
-        'core.entity_view_display.node.d_product.default' => 'c4c770c7f830f5a335f1625c36fb034b', // @todo: Check if this should be here
-        'core.entity_view_display.node.d_product.full' => '6058af0248dddc6eafb22e6938f6c67a', // @todo: Check if this should be here
-        'core.entity_view_display.node.d_product.teaser' => '4457e626eccea06724b2d7532af23738', // @todo: Check if this should be here
-        'core.entity_form_display.node.d_product.default' => 'd9ad0308f61cbe59599fba0b84bb0d06', // @todo: Check if this should be here
+        'core.entity_view_display.node.d_product.d_small_box' => '',
```

Comment:
@rafal-sereda-droptica @piotr-kamieniecki-droptica 
Czy to na 100% powinno być w `droopler_update_8103()` skoro migracja Media jest w późniejszych hookach? Bo pola media wciągamy w `8104` a view modes i form modes w `8105`